npm-publish/bin/publish.sh

```diff
@@ -93,9 +93,9 @@ echo_title "npm ci + test"
 npm ci --ignore-scripts
 
 # Run scripts + tests without auth token to prevent malicious access
-NODE_AUTH_TOKEN= npm rebuild
-NODE_AUTH_TOKEN= npm run prepare --if-present
-NODE_AUTH_TOKEN= npm test
+NODE_AUTH_TOKEN='' npm rebuild
+NODE_AUTH_TOKEN='' npm run prepare --if-present
+NODE_AUTH_TOKEN='' npm test
```

Comment:
[SC1007](https://www.shellcheck.net/wiki/SC1007): Remove space after `=` if trying to assign a value (or for empty string, use `var=''` ... ).